### ISSUE TITLE
Insert updated services

### ISSUE THREAD

**Issue by ClausWiingreen** [Beauty-by-Livia/website#3]
Livia has updated the services so they should be changed in the code aswell

**Comment by Liviabcrw** [comment]
Pyramid really the best. I had made 4 packages but the fourth one is not important it was just to fill space I believe that 3 really is more direct

**Comment by Liviabcrw** [comment]
Pyramid really the best. I had made 4 packages but the fourth one is not important it was just to fill space I believe that 3 really is more direct

**Comment by ClausWiingreen** [comment]
So here is how the layout is currently, i think i might take out some of the common elements of each package to a "All packages includes..." section and then add a small, "why this is a good option" section at the top of each package
![image](https://github.com/user-attachments/assets/9cdb737b-57b7-498e-ab19-ed0be025cf85)

**Comment by Liviabcrw** [comment]
I agree with you about the pyramid. And I also believe it would be less
repetitive if it had the element with all the packages includes.

Just one thing I saw and I believe it was my typing error: the Classic and
Essential packages have the wrong prices. Because what really changes from
one to the other is the trial, and that's why the one that includes the
trial needs to be more expensive. But I can change it in the docs too. I
sent everything you sent me to my friend who is a wedding photographer and
she really liked it too.
